diff --git a/resource/ui/hudmediccharge.res b/resource/ui/hudmediccharge.res
index 6d1dc6a..628c5a8 100644
--- a/resource/ui/hudmediccharge.res
+++ b/resource/ui/hudmediccharge.res
@@ -22,10 +22,10 @@
 		"ControlName"		"CExLabel"
 		"fieldName"			"ChargeLabel"
 		"xpos"				"c-60"
-		"ypos"				"30"
+		"ypos"				"0"
 		"zpos"				"2"
 		"wide"				"120"
-		"tall"				"42"
+		"tall"				"15"
 		"autoResize"		"1"
 		"pinCorner"			"2"
 		"visible"			"1"
@@ -35,19 +35,19 @@
 		"textAlignment"		"center"
 		"dulltext"			"0"
 		"brighttext"		"0"
-		"font"				"FontBold22"
-		"fgcolor"			"White"
+		"font"				"FontBold12"
+		"fgcolor"			"0 255 0 255"
 	}
 
 	"ChargeLabelShadow"
 	{
 		"ControlName"		"CExLabel"
 		"fieldName"			"ChargeLabelShadow"
-		"xpos"				"-2"
-		"ypos"				"-2"
+		"xpos"				"-1"
+		"ypos"				"-1"
 		"zpos"				"1"
 		"wide"				"120"
-		"tall"				"42"
+		"tall"				"15"
 		"autoResize"		"1"
 		"pinCorner"			"2"
 		"visible"			"1"
@@ -57,7 +57,7 @@
 		"textAlignment"		"center"
 		"dulltext"			"0"
 		"brighttext"		"0"
-		"font"				"FontBold22"
+		"font"				"FontBold12"
 		"fgcolor"			"ShadowBlack"
 
 		"pin_to_sibling"		"ChargeLabel"
@@ -111,7 +111,7 @@
 		"fieldName"			"ChargeMeter"
 		"font"				"Default"
 		"xpos"				"0"
-		"ypos"				"-1"
+		"ypos"				"87"
 		"zpos"				"1"
 		"wide"				"120"
 		"tall"				"10"
@@ -260,4 +260,4 @@
 		"scaleImage"		"1"
 		"pin_to_sibling"	"ResistIconAnchor"
 	}
-}
\ No newline at end of file
+}
diff --git a/resource/ui/hudplayerclass.res b/resource/ui/hudplayerclass.res
index e4c5dba..99a4914 100644
--- a/resource/ui/hudplayerclass.res
+++ b/resource/ui/hudplayerclass.res
@@ -10,8 +10,8 @@
 		"zpos"			"1"
 		"wide"			"f0"
 		"tall"			"480"
-		"visible"		"1"
-		"enabled"		"1"
+		"visible"		"0"
+		"enabled"		"0"
 	}
 	
 	"PlayerStatusClassImage"
diff --git a/resource/ui/targetid.res b/resource/ui/targetid.res
index 50712f0..116d71d 100644
--- a/resource/ui/targetid.res
+++ b/resource/ui/targetid.res
@@ -290,4 +290,4 @@
 		"scaleImage"		"1"
 		"color_outline"		"HudTimerProgressInActive"
 	}
-}
\ No newline at end of file
+}
diff --git a/scripts/hudanimations_custom.txt b/scripts/hudanimations_custom.txt
index 83ccf84..b75ccce 100644
--- a/scripts/hudanimations_custom.txt
+++ b/scripts/hudanimations_custom.txt
@@ -59,15 +59,15 @@ event HudHealthDyingPulseStop
 
 event HudLowAmmoPulse
 {
-	Animate	AmmoBG						BgColor		"LowAmmo"			Linear 0.0 0.2
-	Animate	AmmoBG						BgColor		"LowAmmoPulse"		Linear 0.2 0.6
-
-//	Animate	AmmoInClip					FgColor		"LowAmmo"			Linear 0.0 0.2
-//	Animate	AmmoInClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
-//	Animate	AmmoInReserve				FgColor		"LowAmmo"			Linear 0.0 0.2
-//	Animate	AmmoInReserve				FgColor		"LowAmmoPulse"		Linear 0.2 0.6
-//	Animate	AmmoNoClip					FgColor		"LowAmmo"			Linear 0.0 0.2
-//	Animate	AmmoNoClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
+//	Animate	AmmoBG						BgColor		"LowAmmo"			Linear 0.0 0.2
+//	Animate	AmmoBG						BgColor		"LowAmmoPulse"		Linear 0.2 0.6
+
+	Animate	AmmoInClip					FgColor		"LowAmmo"			Linear 0.0 0.2
+	Animate	AmmoInClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
+	Animate	AmmoInReserve				FgColor		"LowAmmo"			Linear 0.0 0.2
+	Animate	AmmoInReserve				FgColor		"LowAmmoPulse"		Linear 0.2 0.6
+	Animate	AmmoNoClip					FgColor		"LowAmmo"			Linear 0.0 0.2
+	Animate	AmmoNoClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
 
 	RunEvent HudLowAmmoPulseLoop		0.4
 }
diff --git a/scripts/hudlayout.res b/scripts/hudlayout.res
index 8b82314..da00d4b 100644
--- a/scripts/hudlayout.res
+++ b/scripts/hudlayout.res
@@ -151,7 +151,7 @@
 		"visible"			"1"
 		"enabled"			"1"
 		"xpos"				"0"
-		"ypos"				"c38"
+		"ypos"				"c15"
 		"wide"				"f0"
 		"tall"				"300"
 	}
@@ -301,7 +301,7 @@
 		"visible" 				"0"
 		"enabled" 				"1"
 		"xpos"					"c-126"
-		"ypos"					"265"
+		"ypos"					"c0"
 		"wide"					"200"
 		"tall"					"50"
 		"priority"				"40"
@@ -325,7 +325,7 @@
 		"visible" 				"0"
 		"enabled" 				"1"
 		"xpos"					"c-126"
-		"ypos"					"355"
+		"ypos"					"c30"
 		"wide"					"200"
 		"tall"					"50"
 		"priority"				"35"
@@ -493,18 +493,18 @@
 		"fieldName" 			"HudDeathNotice"
 		"visible" 				"1"
 		"enabled" 				"1"
-		"xpos"	 				"r635"
+		"xpos"	 				"r875"
 		"ypos"	 				"25"
 		"wide"	 				"628"
 		"tall"	 				"468"
-		"MaxDeathNotices"		"5"
+		"MaxDeathNotices"		"8"
 		"IconScale"				"0.35"
 		"LineHeight"			"12"
 		"LineSpacing"			"0"
 		"CornerRadius"			"3"
 		"RightJustify"			"1"
 		"TextFont"				"FontRegular11"
-		"TextFont_minmode"		"Transparent"
+		//"TextFont_minmode"		"Transparent"
 		"TeamBlue"				"HUDBlueTeamSolid"
 		"TeamRed"				"HUDRedTeamSolid"
 		"IconColor"				"White"
