diff --git a/resource/scheme/clientscheme_colors.res b/resource/scheme/clientscheme_colors.res
index ac9f11d..a9b6f94 100644
--- a/resource/scheme/clientscheme_colors.res
+++ b/resource/scheme/clientscheme_colors.res
@@ -29,10 +29,10 @@ Scheme
 		// Gameplay
 		"Overheal"					"0 170 127 255"
 		"OverhealPulse"				"0 170 127 200"
-		"LowHealth"					"190 20 20 255"
-		"LowHealthPulse"			"190 20 20 200"
-		"LowAmmo"					"190 20 20 255"
-		"LowAmmoPulse"				"190 20 20 200"
+		"LowHealth"					"255 20 20 255"
+		"LowHealthPulse"			"255 20 20 200"
+		"LowAmmo"					"255 20 20 255"
+		"LowAmmoPulse"				"255 20 20 200"
 		"PositiveValue"				"0 170 127 255"
 		"NegativeValue"				"190 20 20 200"
 		"TargetHealth"				"0 170 127 255"
diff --git a/resource/ui/hudammoweapons.res b/resource/ui/hudammoweapons.res
index 3dde886..3c23b18 100644
--- a/resource/ui/hudammoweapons.res
+++ b/resource/ui/hudammoweapons.res
@@ -133,4 +133,4 @@
 		"labelText"			"%Ammo%"
 		"pin_to_sibling"	"AmmoNoClip"
 	}
-}
\ No newline at end of file
+}
diff --git a/resource/ui/huddamageaccount.res b/resource/ui/huddamageaccount.res
index 0a05739..fff763b 100644
--- a/resource/ui/huddamageaccount.res
+++ b/resource/ui/huddamageaccount.res
@@ -8,7 +8,7 @@
 		"delta_item_end_y"		"0"
 		"PositiveColor"			"TargetHealth"
 		"NegativeColor"			""	// Unused
-		"delta_lifetime"		"2"
+		"delta_lifetime"		"4"
 		"delta_item_font"		"FontBold20Outline"
 		"delta_item_font_big"	"FontBold20Outline"
 	}
@@ -17,8 +17,8 @@
 	{
 		"ControlName"			"CExLabel"
 		"fieldName"				"DamageAccountValue"
-		"xpos"					"c-120"
-		"ypos"					"c70"
+		"xpos"					"c-100"
+		"ypos"					"c50"
 		"zpos"					"2"
 		"wide"					"60"
 		"tall"					"26"
diff --git a/resource/ui/huditemeffectmeter.res b/resource/ui/huditemeffectmeter.res
index 3a9cc82..01663e8 100644
--- a/resource/ui/huditemeffectmeter.res
+++ b/resource/ui/huditemeffectmeter.res
@@ -6,7 +6,7 @@
 		"visible"			"1"
 		"enabled"			"1"
 		"xpos"				"c-60"
-		"ypos"				"c120"
+		"ypos"				"c100"
 		"wide"				"120"
 		"tall"				"8"
 		"MeterFG"			"White"
diff --git a/resource/ui/huditemeffectmeter_cleaver.res b/resource/ui/huditemeffectmeter_cleaver.res
index 9030e22..6347c08 100644
--- a/resource/ui/huditemeffectmeter_cleaver.res
+++ b/resource/ui/huditemeffectmeter_cleaver.res
@@ -4,6 +4,6 @@
 {
 	"HudItemEffectMeter"
 	{
-		"ypos"				"c110"
+		"ypos"				"c90"
 	}
 }
diff --git a/resource/ui/huditemeffectmeter_sodapopper.res b/resource/ui/huditemeffectmeter_sodapopper.res
index bfe5616..bf80913 100644
--- a/resource/ui/huditemeffectmeter_sodapopper.res
+++ b/resource/ui/huditemeffectmeter_sodapopper.res
@@ -4,6 +4,6 @@
 {
 	"HudItemEffectMeter"
 	{
-		"ypos"				"c100"
+		"ypos"				"c80"
 	}
 }
diff --git a/resource/ui/hudmediccharge.res b/resource/ui/hudmediccharge.res
index 6d1dc6a..e097121 100644
--- a/resource/ui/hudmediccharge.res
+++ b/resource/ui/hudmediccharge.res
@@ -22,10 +22,10 @@
 		"ControlName"		"CExLabel"
 		"fieldName"			"ChargeLabel"
 		"xpos"				"c-60"
-		"ypos"				"30"
+		"ypos"				"0"
 		"zpos"				"2"
 		"wide"				"120"
-		"tall"				"42"
+		"tall"				"15"
 		"autoResize"		"1"
 		"pinCorner"			"2"
 		"visible"			"1"
@@ -35,19 +35,19 @@
 		"textAlignment"		"center"
 		"dulltext"			"0"
 		"brighttext"		"0"
-		"font"				"FontBold22"
-		"fgcolor"			"White"
+		"font"				"FontBold12"
+		"fgcolor"			"0 255 0 255"
 	}
 
 	"ChargeLabelShadow"
 	{
 		"ControlName"		"CExLabel"
 		"fieldName"			"ChargeLabelShadow"
-		"xpos"				"-2"
-		"ypos"				"-2"
+		"xpos"				"-1"
+		"ypos"				"-1"
 		"zpos"				"1"
 		"wide"				"120"
-		"tall"				"42"
+		"tall"				"15"
 		"autoResize"		"1"
 		"pinCorner"			"2"
 		"visible"			"1"
@@ -57,7 +57,7 @@
 		"textAlignment"		"center"
 		"dulltext"			"0"
 		"brighttext"		"0"
-		"font"				"FontBold22"
+		"font"				"FontBold12"
 		"fgcolor"			"ShadowBlack"
 
 		"pin_to_sibling"		"ChargeLabel"
@@ -111,7 +111,7 @@
 		"fieldName"			"ChargeMeter"
 		"font"				"Default"
 		"xpos"				"0"
-		"ypos"				"-1"
+		"ypos"				"77"
 		"zpos"				"1"
 		"wide"				"120"
 		"tall"				"10"
@@ -260,4 +260,4 @@
 		"scaleImage"		"1"
 		"pin_to_sibling"	"ResistIconAnchor"
 	}
-}
\ No newline at end of file
+}
diff --git a/resource/ui/hudplayerclass.res b/resource/ui/hudplayerclass.res
index e4c5dba..99a4914 100644
--- a/resource/ui/hudplayerclass.res
+++ b/resource/ui/hudplayerclass.res
@@ -10,8 +10,8 @@
 		"zpos"			"1"
 		"wide"			"f0"
 		"tall"			"480"
-		"visible"		"1"
-		"enabled"		"1"
+		"visible"		"0"
+		"enabled"		"0"
 	}
 	
 	"PlayerStatusClassImage"
diff --git a/resource/ui/hudplayerhealth.res b/resource/ui/hudplayerhealth.res
index a361646..a547dfc 100644
--- a/resource/ui/hudplayerhealth.res
+++ b/resource/ui/hudplayerhealth.res
@@ -5,8 +5,8 @@
 	{
 		"ControlName"	"EditablePanel"
 		"fieldName"		"HudPlayerHealth"
-		"xpos"			"c-190"
-		"ypos"			"c68"
+		"xpos"			"c-170"
+		"ypos"			"c48"
 		"zpos"			"2"
 		"wide"			"150"
 		"tall"			"150"
diff --git a/resource/ui/targetid.res b/resource/ui/targetid.res
index 50712f0..116d71d 100644
--- a/resource/ui/targetid.res
+++ b/resource/ui/targetid.res
@@ -290,4 +290,4 @@
 		"scaleImage"		"1"
 		"color_outline"		"HudTimerProgressInActive"
 	}
-}
\ No newline at end of file
+}
diff --git a/scripts/hudanimations_custom.txt b/scripts/hudanimations_custom.txt
index 83ccf84..35e526f 100644
--- a/scripts/hudanimations_custom.txt
+++ b/scripts/hudanimations_custom.txt
@@ -1,11 +1,11 @@
 // Bonus Health Pulse
 event HudHealthBonusPulse
 {
-	Animate	HealthBG					BgColor		"Overheal"			Linear 0.0 0.2
-	Animate	HealthBG					BgColor		"OverhealPulse"		Linear 0.2 0.6
+//	Animate	HealthBG					BgColor		"Overheal"			Linear 0.0 0.2
+//	Animate	HealthBG					BgColor		"OverhealPulse"		Linear 0.2 0.6
 
-//	Animate	PlayerStatusHealthValue		FgColor		"Overheal"			Linear 0.0 0.2
-//	Animate	PlayerStatusHealthValue		FgColor		"OverhealPulse"		Linear 0.2 0.6
+	Animate	PlayerStatusHealthValue		FgColor		"Overheal"			Linear 0.0 0.2
+	Animate	PlayerStatusHealthValue		FgColor		"OverhealPulse"		Linear 0.2 0.6
 
 	Animate	GuiPlayerHealth				FgColor		"PositiveValue"		Linear 0.0 0.0
 	Animate RedPlayerHealth				FgColor		"PositiveValue"		Linear 0.0 0.0
@@ -30,11 +30,11 @@ event HudHealthBonusPulseStop
 // Low Health Pulse
 event HudHealthDyingPulse
 {
-	Animate	HealthBG					BgColor		"LowHealth"			Linear 0.0 0.2
-	Animate	HealthBG					BgColor		"LowHealthPulse"	Linear 0.2 0.6
+//	Animate	HealthBG					BgColor		"LowHealth"			Linear 0.0 0.2
+//	Animate	HealthBG					BgColor		"LowHealthPulse"	Linear 0.2 0.6
 
-//	Animate	PlayerStatusHealthValue		FgColor		"LowHealth"			Linear 0.0 0.2
-//	Animate PlayerStatusHealthValue		FgColor		"LowHealthPulse"	Linear 0.2 0.6
+	Animate	PlayerStatusHealthValue		FgColor		"LowHealth"			Linear 0.0 0.2
+	Animate PlayerStatusHealthValue		FgColor		"LowHealthPulse"	Linear 0.2 0.6
 
 	Animate	GuiPlayerHealth				FgColor		"NegativeValue"		Linear 0.0 0.0
 	Animate RedPlayerHealth				FgColor		"HudWhite"			Linear 0.0 0.0
@@ -59,15 +59,15 @@ event HudHealthDyingPulseStop
 
 event HudLowAmmoPulse
 {
-	Animate	AmmoBG						BgColor		"LowAmmo"			Linear 0.0 0.2
-	Animate	AmmoBG						BgColor		"LowAmmoPulse"		Linear 0.2 0.6
+//	Animate	AmmoBG						BgColor		"LowAmmo"			Linear 0.0 0.2
+//	Animate	AmmoBG						BgColor		"LowAmmoPulse"		Linear 0.2 0.6
 
-//	Animate	AmmoInClip					FgColor		"LowAmmo"			Linear 0.0 0.2
-//	Animate	AmmoInClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
-//	Animate	AmmoInReserve				FgColor		"LowAmmo"			Linear 0.0 0.2
-//	Animate	AmmoInReserve				FgColor		"LowAmmoPulse"		Linear 0.2 0.6
-//	Animate	AmmoNoClip					FgColor		"LowAmmo"			Linear 0.0 0.2
-//	Animate	AmmoNoClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
+	Animate	AmmoInClip					FgColor		"LowAmmo"			Linear 0.0 0.2
+	Animate	AmmoInClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
+	Animate	AmmoInReserve				FgColor		"LowAmmo"			Linear 0.0 0.2
+	Animate	AmmoInReserve				FgColor		"LowAmmoPulse"		Linear 0.2 0.6
+	Animate	AmmoNoClip					FgColor		"LowAmmo"			Linear 0.0 0.2
+	Animate	AmmoNoClip					FgColor		"LowAmmoPulse"		Linear 0.2 0.6
 
 	RunEvent HudLowAmmoPulseLoop		0.4
 }
diff --git a/scripts/hudlayout.res b/scripts/hudlayout.res
index 8b82314..0dec39d 100644
--- a/scripts/hudlayout.res
+++ b/scripts/hudlayout.res
@@ -100,8 +100,8 @@
 		"fieldName"			"HudWeaponAmmo"
 		"visible"			"1"
 		"enabled"			"1"
-		"xpos"				"c90"
-		"ypos"				"c93"
+		"xpos"				"c70"
+		"ypos"				"c73"
 		"wide"				"110"
 		"tall"				"43"
 	}
@@ -151,7 +151,7 @@
 		"visible"			"1"
 		"enabled"			"1"
 		"xpos"				"0"
-		"ypos"				"c38"
+		"ypos"				"c15"
 		"wide"				"f0"
 		"tall"				"300"
 	}
@@ -301,7 +301,7 @@
 		"visible" 				"0"
 		"enabled" 				"1"
 		"xpos"					"c-126"
-		"ypos"					"265"
+		"ypos"					"c0"
 		"wide"					"200"
 		"tall"					"50"
 		"priority"				"40"
@@ -325,7 +325,7 @@
 		"visible" 				"0"
 		"enabled" 				"1"
 		"xpos"					"c-126"
-		"ypos"					"355"
+		"ypos"					"c30"
 		"wide"					"200"
 		"tall"					"50"
 		"priority"				"35"
@@ -493,18 +493,18 @@
 		"fieldName" 			"HudDeathNotice"
 		"visible" 				"1"
 		"enabled" 				"1"
-		"xpos"	 				"r635"
+		"xpos"	 				"c-314"
 		"ypos"	 				"25"
 		"wide"	 				"628"
 		"tall"	 				"468"
-		"MaxDeathNotices"		"5"
+		"MaxDeathNotices"		"8"
 		"IconScale"				"0.35"
 		"LineHeight"			"12"
 		"LineSpacing"			"0"
 		"CornerRadius"			"3"
 		"RightJustify"			"1"
 		"TextFont"				"FontRegular11"
-		"TextFont_minmode"		"Transparent"
+		//"TextFont_minmode"		"Transparent"
 		"TeamBlue"				"HUDBlueTeamSolid"
 		"TeamRed"				"HUDRedTeamSolid"
 		"IconColor"				"White"
